praetorian_cli/sdk/entities/agents.py

```
@@ -36,6 +39,97 @@ def affiliation_filename(self, agent_type: str, key: str) -> str:
+                    last_key = msg.get('key', '')
+
+                    if role == 'chariot':
+                        return {
+                            'response': content,
+                            'conversation_id': conversation_id,
+                            'tool_calls': tool_calls,
+                        }
+                    elif role in ('tool call', 'tool response'):
+                        tool_calls.append({'role': role, 'content': content})
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)